### DISCUSSION TITLE
Not working since yesterday

### DISCUSSION THREAD

**Discussion by steffonix** [shaked6540/YoutubePlaylistDownloader#355] [Q&A]
Thank you very much for your very good work. I would just like to point out that YTPD has not been working for me in France for a few hours. Neither for video content nor for audio content. Kind regards. Steff

**Comment by whtmst** [comment]
Hi, I can also confirm that it stopped working in Azerbaijan 4 days ago. No video or playlist from YouTube is downloading.
Here is the error code when trying to download any video:

Here is a list of the videos that could not be downloaded, and the reason why: 
Billx x Galvanoize x Etno Skupina Čuvarice - Ruza Za Drugoga (rave edit) Reason: Response status code does not indicate success: 403 (Forbidden).
The Flute Tune (Soulpride Remix) Reason: Response status code does not indicate success: 403 (Forbidden).
Expecto Patronum -  EDM  House Harry Potter MashUp by Mashd n Kutcher Reason: Response status code does not indicate success: 403 (Forbidden).
KSHMR & 22Bullets - Devotion (Official Video) Reason: Response status code does not indicate success: 403 (Forbidden).

**Reply by helloidontknoww** [reply]
I had the same problem

**Reply by helloidontknoww** [reply]
and this in my discussion: [v](https://github.com/shaked6540/YoutubePlaylistDownloader/discussions/367)

**Reply by Empty23** [reply]
Have you solve this error?

**Comment by yccjinfo** [comment]
This lousy Israeli programmer had integrated a hidden spyware in his App to collect passwords stored in the computer.
DO NOT INSTALL HIS APP AND REPORT HIM! FUCK ISRAEL AND ISRAELI CHILDREN KILLERS EVERYWHERE

**Reply by whtmst** [reply]
Excuse me, but is there any way to see proof of your claims? It's easy to write anything, but without proof, such accusations are worthless. I want to say upfront that I'm not defending the author. I just want to be sure that what you're writing is true, and if it is, I'll gladly delete the program.

**Reply by shaked6540** [reply]
He spammed every issue and discussion on the repo with his copy paste message, I'm working to delete everything

There are no spyware in the software, its open source..

**Reply by dwsiddall** [reply]
Thank you for this clarification.
